<?xml version="1.0"?>
<launch>
  <node name="position_control_node" pkg="bebop_pos_ctrl" type="position_control_node" output="screen" >
        <param name="Hz" value="10" />
        <param name="time_hover" value="100000" />
        <param name="debug" value="false" />

        <param name="K_p_x" value="0.07" />
        <param name="K_i_x" value="0.0" />
        <param name="K_d_x" value="0.0" />

        <param name="K_p_y" value="0.07" />
        <param name="K_i_y" value="0.0" />
        <param name="K_d_y" value="0.0" />

        <param name="MaxV_xy" value="0.5" />  
        <param name="Limit_xy_error_int" value="0.1" />
        <param name="Tolerance_hori_pos" value="0.1" />

        <param name="K_p_z" value="0.5" />
        <param name="K_i_z" value="0.0" />
        <param name="K_d_z" value="0.0" />
        <param name="MaxV_z" value="1.0" />
        <param name="Limit_z_error_int" value="0.1" />
        <param name="Tolerance_vert_pos" value="0.05" />

        <param name="K_p_yaw" value="0.7" />
        <param name="K_i_yaw" value="0.0" />
        <param name="K_d_yaw" value="0.0" />
        <param name="MaxV_yaw" value="1.0"/>
        <param name="Limit_yaw_error_int" value="0.3"/>
        <param name="Tolerance_yaw_rad" value="0.1"/>
  </node>
</launch>
